src/library/scala/concurrent/Future.scala

```diff
@@ -243,7 +243,17 @@ trait Future[+T] extends Awaitable[T] {
    *  this future. If this future is completed with an exception then the new
    *  future will also contain this exception.
    *
-   *  $forComprehensionExamples
+   *  Example:
+   *
+   *  {{{
+   *  val f = Future { "The future" }
+   *  val g = f map { x: String => x + " is now!" }
+   *  }}}
+   *
+   *  Note that a for comprehension involving a {{{Future}}} 
+   *  may expand to include a call to {{{map}}} and or {{{flatMap}}} 
+   *  and {{{withFilter}}}.  See {{{flatMap}}} for an example of such a comprehension.
```

Comment:
Triple `{` are for code blocks, could you please use a single back-ticks to enclose monospace phrases here?

The pointer to the example should also be a link. Here is the syntax for defining a link to a non-overloaded method:

```
/** Example [[scala.concurrent.Future#flatMap]] */
```